exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/first_steps.ipynb

```diff
@@ -256,9 +256,7 @@
    "id": "0dc80445-e5c0-43c1-ae34-f16326d6fb94",
    "metadata": {},
    "source": [
-    "**Please note**: \n",
-    "* Parquet import requires using **<span style=\"color: #40a\">Exasol version 2025.\\* or higher</span>**, see [docs.exasol.com](https://docs.exasol.com/db/latest/loading_data/load_data_parquet.htm).\n",
-    "* Hence, the import is currently commented out.\n",
+    "**Please note**: Parquet import requires using **<span style=\"color: #40a\">Exasol version 2025.\\* or higher</span>**, see [docs.exasol.com](https://docs.exasol.com/db/latest/loading_data/load_data_parquet.htm).\n",
```

Comment:
Might be good to clarify somewhere in this notebook. I don't know if here is the right spot.

The PyExasol variant using import_* and export_* with a local file system should work for 7.x, 8.x, & 2025.x, but using the the IMPORT query command for a remote parquet file only works fro 2025.x

[The PyExasol variant works as we transform the local parquet file to a streamed CSV. Thus, it's not dependent on the DB changes.]